src/content/docs/en/guides/images.mdx

```diff
@@ -700,3 +700,24 @@ If you were using the image integration in Astro v2.x, complete the following st
 You can now declare an associated image for a content collections entry, such as a blog post's cover image, in your frontmatter using its path relative to the current folder.
 
 The new `image` helper for content collections lets you validate the image metadata using Zod. Learn more about [how to use images in content collections](/en/guides/images/#images-in-content-collections)
+
+### Navigating Image Imports in Astro v3.0
+
+In Astro v3.0, if you find yourself wanting to preserve the old import behavior for images and require a string representation of the image's URL, consider appending `?raw` or `?url` to the end of your image path when importing it. For example: 
```

Comment:
The `?raw` syntax does not return a string representation of the image's URL, when used with image formats like `.png`, `.webp`, etc it will return binary
```suggestion
In Astro v3.0, if you find yourself wanting to preserve the old import behavior for images and require a string representation of the image's URL, consider appending `?url` to the end of your image path when importing it. For example: 
```

Reply:
woops, my bad. I actually both were meant for the same purpose, I will apply this change asap.